pulseapi/entries/views.py

```diff
@@ -149,7 +149,7 @@ class EntriesListView(ListCreateAPIView):
 
     #Query Parameters -
 
-    - `?search=` - Allows search terms
+    - `?search=` - Search by title, description, creator, or tag.
```

Comment:
Based on mattermost chat, this should be "Search based on title, description, creators, *and* tags" rather than *or*, since users don't have a choice here: all fields will be checked.

Based on that description though, if we're searching in creators, are we missing publisher in this list? They're effectively the same category of data.